### ISSUE TITLE
Fix network connection. 

### ISSUE THREAD

**Issue by psykiatris** [psykiatris/Sweet-Potato-MUD#1]
When running program, it appears to connect to local host, but doesn’t. Changing port to different ones has no effect.

**Comment by psykiatris** [comment]
In debug sessions, a connection appears to be made that is not reflected in the frame. So, the connection is open. 

Also, if a co-op is chosen, it should mean a person on the same network, however it crashes saying that port is already in use. 

This whole thing probably should be re-coded to create server whether local or no.

**Comment by psykiatris** [comment]
I was finally able to run the app. I had to run it and select 1. I left it running. 

Then I opened a Ecco day terminal window, ran it afain, also choosing 1. I was able to create a user with a password and was put into the starter area. Then it stopped working. 

I was getting exception through the console, probably because I compiled it under Java 10. 

I need to consider a few options. 1. Keep it Java 8. 2. Split the app so there will be a server app and a client app. 

Also it crashes when choosing co-op mode, as the same port cannot be accessed at the same time. 

I’m examining the thread and concurrent packages with the hope of migrating it to Java10 (or higher).
